app/views/templates/Layout.scala.html

```diff
@@ -52,19 +52,33 @@
   <link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="all" rel="stylesheet" type="text/css">
   <link href="@routes.Assets.versioned("stylesheets/accessible-autocomplete.min.css").url" media="screen" rel="stylesheet" type="text/css" />
 
-    @draftId.map { id =>
+    @if(draftId.isDefined) {
         @hmrcTimeoutDialogHelper(
             timeout             = Some(appConfig.timeout),
             countdown           = Some(appConfig.countdown),
-            keepAliveUrl        = Some(routes.KeepAliveController.keepAlive(id).url),
+            keepAliveUrl        = Some(routes.KeepAliveController.keepAlive(draftId.get).url),
             keepAliveButtonText = Some(messages("timeout.keepAlive")),
             signOutUrl          = controllers.auth.routes.AuthController.signOut().url,
             signOutButtonText   = Some(messages("timeout.signOut")),
             timeoutUrl          = Some(controllers.auth.routes.AuthController.signOutNoSurvey().url),
             title               = Some(messages("timeout.title")),
             message             = Some(messages("timeout.message"))
         )
-    }.getOrElse(additionalHeadBlock)
+    } else {
+        @hmrcTimeoutDialogHelper(
+            timeout             = Some(appConfig.timeout),
+            countdown           = Some(appConfig.countdown),
+            keepAliveUrl        = None,
+            keepAliveButtonText = Some(messages("timeout.keepAlive")),
+            signOutUrl          = controllers.auth.routes.AuthController.signOut().url,
+            signOutButtonText   = Some(messages("timeout.signOut")),
+            timeoutUrl          = Some(controllers.auth.routes.AuthController.signOutNoSurvey().url),
+            title               = Some(messages("timeout.title")),
+            message             = Some(messages("timeout.message"))
+        )
+        @additionalHeadBlock
+    }
+
```

Comment:
DRY - lets towel this off - you can get the same result by removing the if statement and using a fold on draftId for keepAliveUrl